src/routers/healthCheck.js

```diff
@@ -11,4 +12,12 @@ router.get('/_healthcheck', (req, res) => {
   res.status(200).send('EUREKA!');
 });
 
+router.post('/login', authController.login);
```

Comment:
I don't really like the idea of controllers in the express archictecture as it is a middleware-based framework, as I see it would make more sense to create routers for each of the resources (/login, /logout in this case)